portfolio.html

```diff
@@ -0,0 +1,96 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+	<title>Max G Portfolio</title>
+	<link rel="Stylesheet" type="text/css" href="portfolio.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+	
+	<header class="bg-image">
+		
+		<nav class="navbar">
+			<div class="logo-div">
+				<img class="logo" src="https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/089641db-8d3d-40f6-92e8-1744e0272fed/db18sam-17538ca1-c20f-43c2-aeec-e74871345a14.png/v1/fill/w_1024,h_820,strp/clean_wolf_logo_by_akther_brothers__no_watermark__by_shoaibakther_db18sam-fullview.png?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9ODIwIiwicGF0aCI6IlwvZlwvMDg5NjQxZGItOGQzZC00MGY2LTkyZTgtMTc0NGUwMjcyZmVkXC9kYjE4c2FtLTE3NTM4Y2ExLWMyMGYtNDNjMi1hZWVjLWU3NDg3MTM0NWExNC5wbmciLCJ3aWR0aCI6Ijw9MTAyNCJ9XV0sImF1ZCI6WyJ1cm46c2VydmljZTppbWFnZS5vcGVyYXRpb25zIl19.bQPKV239OIZBAPo4MBe1UlZ6l4St10WXzg03HogOS98" alt="Max G Logo">
```

Comment:
worth downloading this and adding it to git. If this is not hosted where you expect it, the link will break and the page will look bad.